extruct/jsonld.py

```diff
@@ -28,14 +28,23 @@ def extract_items(self, document, base_url=None):
             if items for item in items if item
         ]
 
+    def _may_be_get_json(self, script):
+        try:
+            return json.loads(script, strict=False)
+        except Exception:
+            return False
+
     def _extract_items(self, node):
         script = node.xpath('string()')
-        try:
-            # TODO: `strict=False` can be configurable if needed
-            data = json.loads(script, strict=False)
-        except ValueError:
-            # sometimes JSON-decoding errors are due to leading HTML or JavaScript comments
```

Comment:
I'd rather not remove this comment, I think it's useful

Reply:
Added the comment back to the code.